src/libraries/Common/tests/System/IO/Compression/CompressionStreamUnitTestBase.cs

```diff
@@ -14,7 +14,7 @@ public abstract class CompressionStreamUnitTestBase : CompressionStreamTestBase
     {
         private const int TaskTimeout = 30 * 1000; // Generous timeout for official test runs
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
Assuming the issue here is the blocking, these tests could be made to work; there just wasn't a reason to avoid the blocking previously.

How do we want to track the difference between "this test can never work with wasm" and "this test doesn't currently work with wasm but could be made to"?

Reply:
My plan was to do a pass over all the `PlatformDetection.IsThreadingSupported` items once we have more capacity to actually analyze/rewrite the tests.